templates/blog/article.html

```diff
@@ -125,70 +125,29 @@ <h1>{{ article.title.rendered|safe }}</h1>
           <h2 class="p-muted-heading">Newsletter signup</h2>
         </div>
         <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_1212" onsubmit="ga('send', 'Newsletter', 'Signup', 'New homepage newsletter signup');">
-          <p >Select topics you're interested in:</p>
-
-          <ul class="p-list">
-            <li class="p-list__item u-clearfix">
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="insightscloudserver" name="insightscloudserver" value="true" />
-                <span class="p-checkbox__label" id="insightscloudserver">Cloud and Server</span>
-              </label>
-            </li>
-            <li class="p-list__item u-clearfix">
-              <label class="p-checkbox">
-                <input class="p-checkbox__input" type="checkbox" aria-labelledby="insightsdesktop" name="insightsdesktop" value="true" />
-                <span class="p-checkbox__label" id="insightsdesktop">Desktop</span>
-              </label>
-            </li>
-            <li class="p-list__item u-clearfix">
+            <p>Get the latest Canonical news and updates in your inbox.</p>
```

Comment:
This would've be good to flag in the qa instructions, there's no mention of checking individual articles in addition to the blog page